drivers/SmartThings/zigbee-switch/profiles/wallhero-scene-switch.yml

```diff
@@ -0,0 +1,12 @@
+name: wallhero-scene-switch
```

Comment:
I'd just call this something like `button`

Reply:
> I'd just call this something like `button`

Got it. I will rename wallhero-scene-switch to button. (Both the filename and name component)